components/create_team/components/team_url/team_url.tsx

```diff
@@ -123,11 +134,11 @@ export default class TeamUrl extends React.PureComponent {
         }
 
         this.setState({isLoading: true});
-        var teamSignup = JSON.parse(JSON.stringify(this.props.state));
+        const teamSignup = JSON.parse(JSON.stringify(this.props.state));
         teamSignup.team.type = 'O';
         teamSignup.team.name = name;
 
-        const {exists} = await checkIfTeamExists(name);
+        const {exists}: any = await checkIfTeamExists(name);
```

Comment:
This should be `{exists: boolean}` instead of `any`.

Reply:
It causes the problem that exists is not visible in if statement on 143 line and it does not exist on type ActionFunc.

Reply:
I think you might want to change the return type of `checkIfTeamExists` in your type declaration. It could just look something like: `Promise<{exists: boolean}>`.